scripts/config

```diff
@@ -144,6 +148,14 @@ apply_config() {
     if [ "$fpm_usage" != "$old_fpm_usage" ] || [ "$fpm_footprint" != "$old_fpm_footprint" ]
     then
         ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint
+
+        user=$(ynh_app_setting_get --app=$app --key=user)
+        # use $user instead of $app as user that run the fpm processes
+        finalphpconf="/etc/php/7.0/fpm/pool.d/$app.conf"
+        ynh_replace_string --match_string="^user = .*" --replace_string="user = $user" --target_file="$finalphpconf"
```

Comment:
Shouldn't we patch `ynh_add_fpm_config` instead? (However, we must make sure we have created the user).

Reply:
That's a very specific use case, don't feel like the helper should be modified just for that.

Reply:
I have to admit, I don't get why you patch the conf here.

Reply:
To keep the compatibility with the code https://github.com/YunoHost-Apps/my_webapp_ynh/blob/testing/scripts/install#L60
Even if I don't think that's really useful.

Reply:
And... Why we don't simply use `$app` instead of `webapp${app_nb}`?

Reply:
Because the app was built that way, so we would have to merge all existing app to a new user.
Considering the app is the empty webapp, it would means also ask to all the user to so manually.

Reply:
`$app` contains the app name and the app ID, if we just change it during the install and don't change anything in upgrade, everything should work as expected (but for retro-compatibility we must keep your patch indeed).

Reply:
Yes indeed we could, but only into the install script.
While keeping everything else with $user, even though $user could be $app.

And so, does it really worth it ?

Reply:
No, it was to be sure, and understand that line/choice/quantum physics/life etc

Reply:
If it was only me, considering the difference between $user and $app, I would have kept $app...